CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Empty text or numeric fields are always displayed as `-` in the detail sections of the viewer mode.
 - Add workgroup filter to viewer.
 - Removed unused table `bdms.geodin_files` from database.
+- Replaced all generic browser alerts with styled alerts.
```

Comment:
REQ: Da diese Version bereits released ist muss dieser Text in den nächsten Release (= `[Unreleased]`) verschoben werden.